test/permits/applications/special-uses/commercial/outfitters/outfitters.js

```diff
@@ -0,0 +1,1400 @@
+/*
+
+  ___ ___       ___               _ _       _   ___ ___ 
+ | __/ __|  ___| _ \___ _ _ _ __ (_) |_    /_\ | _ \_ _|
+ | _|\__ \ / -_)  _/ -_) "_| "  \| |  _|  / _ \|  _/| | 
+ |_| |___/ \___|_| \___|_| |_|_|_|_|\__| /_/ \_\_| |___|
+
+*/
+
+//*******************************************************************
+
+'use strict';
+
+//*******************************************************************
+
+const include = require('include')(__dirname);
+
+//*******************************************************************
+
+const request = require('supertest');
+const server = include('index.js');
+const util = include('test/utility.js');
+
+const factory = require('unionized');
+const tempOutfittersInput = include('test/data/testInputOutfitters.json');
+
+const chai = require('chai');
+const expect = chai.expect;
+
+//*******************************************************************
+//Mock Input
+
+const tempOutfittersFactory = factory.factory(tempOutfittersInput);
+
+//*******************************************************************
+
+describe('outfitters POST: validate required fields present', function(){
+
+	let token;
+
+	before(function(done) {
+
+		util.getToken(function(t){
+
+			token = t;
+			return done();
+
+		});
+	
+	});
+
+	describe('body fields', function(){
+
+		it('should return valid json with a 400 status code for outfitters POST request without a body', function(done) {
+		
+			request(server)
+				.post('/permits/applications/special-uses/commercial/outfitters')
+				.set('x-access-token', token)
+				.attach('guideDocumentation', './test/data/test_guideDocumentation.txt')
```

Comment:
Would be good to have these provided by the factory as well, that way there aren't the same 5 lines again and again.